src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (numbers[i] > absMax) {
                 absMax = numbers[i];
             }
         }
-        return absMax;
+        return Math.abs(absMax);
```

Comment:
**Performance Issue**: The change from initializing 'absMax' with the first element of 'numbers' array to 0 and removing the absolute value comparison in the condition check could lead to incorrect results for arrays containing all negative numbers. This approach fails to find the absolute maximum if the highest value is negative, as it starts from 0 and only positive numbers will update 'absMax'. <br> **Fix**: Initialize 'absMax' with the first element of the array and compare the absolute values of 'numbers[i]' and 'absMax' to ensure the correct absolute maximum is found regardless of the sign of the numbers. <br> **Code Suggestion**: 
 ```
 -        int absMax = 0;
 +        int absMax = numbers[0];
 -            if (numbers[i] > absMax) {
 +            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
 ```